deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -224,17 +224,9 @@ def post(self, request, name, stage):
             cluster_name = env.get('clusterName')
             cluster_info = json.loads(request.body)
             log.info("Update Cluster Configuration with {}", cluster_info)
-            cluster_name = '{}-{}'.format(name, stage)
-            current_cluster = clusters_helper.get_cluster(request, cluster_name)
-            log.info("getting current Cluster Configuration is {}", current_cluster)
-            if 'spiffe_id' not in current_cluster and 'spiffe_id' in cluster_info:
-                log.error("Teletraan does not support user to pass in customized spiffe_id %s if exising cluster does not have" % cluster_info['spiffe_id'])
-                raise TeletraanException("Teletraan does not support user to pass in customized spiffe_id")
-            if 'spiffe_id' in current_cluster and 'spiffe_id' in cluster_info:
-                if current_cluster['spiffe_id'] != cluster_info['spiffe_id']:
-                    log.error("Teletraan does not support user to change spiffe_id %s , current spiffe_id %s"
-                        % (cluster_info['spiffe_id'], current_cluster['spiffe_id']))
-                    raise TeletraanException("Teletraan does not support user to change spiffe_id")
+            if 'spiffe_id' in cluster_info:
```

Comment:
did you test the case that a normal update cluster configuration?  i thought in the normal case, the unchanged spiffe_id is also passed into the cluster_info and is required.

Reply:
good catch, thanks. Please review updated change.